examples/integrations/oci/pom.xml

```diff
@@ -38,6 +38,7 @@
         <module>metrics</module>
         <module>objectstorage</module>
         <module>objectstorage-cdi</module>
+        <module>oci-secrets-mp-config-source</module>
```

Comment:
This should be under examples/microprofile and should be named `oci-secrets-config-source`.

Reply:
Looping in @barchetta who advised me on the current location, and @tomas-langer who probably has an opinion. (I don't care.)

Reply:
The `oci-tls-certificates` is already under examples/microprofile.  My recommendation is to make this new example right next to it, and drop the "mp" in the same for consistency with everything else there.

Reply:
This example is not really a Microprofile example - it is an example just using MP Config and nothing else. 
I think that if we want to show how to use this in Helidon MP, it should be a full MP example if a JAX-RS resource injecting something from configuration, rather than a programmatic example using config only...

Reply:
Given the new requirements, I will close this PR and undertake a new one after discussing them with you.

Reply:
There was no need to close the PR - I think a quick slack or zoom could resolve this quickly.